File: android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/rsp/RspLogin.kt
/*
 * Copyright (c) TIKI Inc.
 * MIT license. See LICENSE file in root directory.
 */

package com.mytiki.sdk.capture.receipt.capacitor.rsp

import org.json.JSONObject

class RspLogin(
    private val username: String,
    private val provider: String
) : Rsp {
    override fun toJson(): JSONObject =
        JSONObject()
            .put("username", username)
            .put("provider", provider)
}

File: ios/Plugin/Rsp/RspLogin.swift
/*
 * Copyright (c) TIKI Inc.
 * MIT license. See LICENSE file in root directory.
 */

import Foundation
import Capacitor

struct RspLogin : Rsp {
    private let username: String
    private let provider: String
    
    func toJson() -> JSObject {
        JSObject.updateValue("username", username)
        JSObject.updateValue("provider", provider)
    }
    
}
